templates/flavors/base/linodeVPC.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
+kind: LinodeVPC
+metadata:
+  name: ${VPC_NAME:=${CLUSTER_NAME}}
+spec:
+  region: ${LINODE_REGION}
+  subnets:
+    - ipv4: 10.0.0.0/8
+      label: default
```

Comment:
can we add the following label here: `cluster.x-k8s.io/cluster-name: ${CLUSTER_NAME}`

This is needed to make sure we know what cluster is associated with a VPC and will help us trigger a reconcile loop for VPC when a cluster is paused and unpaused during `clusterctl move` operation.

Reply:
Thanks, updated and added the label.